changelog_page.php

```diff
@@ -241,7 +241,7 @@ function print_project_header_changelog( $p_project_name ) {
 	$t_resolved = config_get( 'bug_resolved_status_threshold' );
 
 	# grab version info for later use
-	$t_version_rows = version_get_all_rows( $t_project_id, null, false );
+	$t_version_rows = version_get_all_rows( $t_project_id, /* released */ null, /* obsolete */ false );
```

Comment:
Instead of cluttering the code with inline comments describing each parameter, it would be better to have
```
# grab released versions info for later use, excluding obsolete ones
```